package-system/Qt/copy_platform_cmakes.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import glob
+import os
+import pathlib
+import platform
+import shutil
+import sys
+
+
+# There are some additional cmake files we need to copy per-platform to the install directory
+platform_system = platform.system().lower()
+platform_to_pal = {
+    "windows": "Windows",
+    "linux": "Linux",
+    "darwin": "Mac"
+}
+
+if not platform_system in platform_to_pal:
```

Comment:
uber nit: `if platform_system not in platform_to_pal:` is slightly more idiomatic

Reply:
Good call, will fix!